Extension/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 ### New Features
 * Add an 'Extract to Function' (or Member Function) code action after selecting code. [#1162](https://github.com/microsoft/vscode-cpptools/issues/1162)
   * Currently, it's only enabled when `C_Cpp.experimentFeatures` is `true`. Also, 'Extract to Free Function' is disabled.
-* Add the option to install a compiler, accessible through the walkthrough. [PR #11286](https://github.com/microsoft/vscode-cpptools/pull/11286)
+* Add an option to install a compiler, accessible through the walkthrough and the Select IntelliSense Configuration experience. [#10525](https://github.com/microsoft/vscode-cpptools/issues/10525)
```

Comment:
We noticed that it isn't accessible in Select IntelliSense Configuration on Windows. Do you want to be more concise about that?  :)

Reply:
I don't think we want to go into all the specifics in the changelog.  How about I shorten it to "Compiler acquisition improvements" and update the Issue to make sure it covers the full behvior?